Makefile

```diff
@@ -386,10 +387,7 @@ build-windows:
 	docker tag antrea/antrea-windows:$(DOCKER_IMG_VERSION) antrea/antrea-windows
 
 .PHONY: build-ubuntu-coverage
-build-ubuntu-coverage:
-	@echo "===> Building Antrea bins and antrea/antrea-ubuntu-coverage Docker image <==="
-	docker build -t antrea/antrea-ubuntu-coverage:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.coverage $(DOCKER_BUILD_ARGS) .
-	docker tag antrea/antrea-ubuntu-coverage:$(DOCKER_IMG_VERSION) antrea/antrea-ubuntu-coverage
+build-ubuntu-coverage: build-controller-ubuntu-coverage build-agent-ubuntu-coverage
```

Comment:
I think that we can remove this now, because it is called only during test from `build-antrea-linux-all.sh`, and there we are already calling the `build-agent-ubuntu` and `build-controller-ubuntu`

Reply:
I have a comment to the Makefile explaining that I am keeping the targets for backwards compatibility. It's just an alias and someone / a script outside of the repo may be using the target directly?